packages/video_player/video_player/lib/video_player.dart

```diff
@@ -5,34 +5,16 @@
 import 'dart:async';
 import 'dart:io';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/material.dart';
 import 'package:meta/meta.dart';
 
-final MethodChannel _channel = const MethodChannel('flutter.io/videoPlayer')
-  // This will clear all open videos on the platform when a full restart is
-  // performed.
-  ..invokeMethod<void>('init');
+import 'package:video_player_platform_interface/video_player_platform_interface.dart';
```

Comment:
You will need to split out the changes to `package:video_player` (besides the simple directory move/version bump) into a follow-up PR because `video_player_platform_interface` can't be uploaded to pub until this PR lands

Reply:
I am confused. There now is one PR for the platform interface and the move. This one to  make the Android and iOS part use the platform interface. And one to add a new web implementation. Do I need to split the first PR into the move and platform interface parts?

Reply:
The first PR needs to just add the `video_player_platform_interface` package. As you can see, the tests are failing because you can't have the `video_player` package depend on `video_player_platform_interface` until after you land `video_player_platform_interface` and it's uploaded to pub.

Reply:
Oops! I am the one who's confused. I thought this was the same PR as before.

Reply:
https://github.com/flutter/plugins/pull/2273 does only add the platform interface. Tests are green. Only https://github.com/flutter/plugins/pull/2276 and https://github.com/flutter/plugins/pull/2279 depend on the interface.

Reply:
No worries. It really is confusing as they all contain mostly the same commits. Plan was to rebase and even squash if that is on as soon as the first one is merged. The other two PRs need some more tests before they can be merged I guess.